### ISSUE TITLE
`Proxied` param is not being forwarded to createDNS request

### ISSUE THREAD

**Issue by sielay** [cloudflare/node-cloudflare#13]
```javascript
client.addDNS(CFClient.DNSRecord.create(
{ zone_id: 'ZONEID',
  type: 'CNAME',
  name: 'SUBDOMAIN',
  ttl: 120,
  proxiable: true,
  proxied: true,
  content: 'TARGET' }
);
```

Results in

```javascript
{
protocol: 'https:',
  path: '/client/v4/zones/ZONEID/dns_records',
  retries: [Function: backoff],
  slashes: true,
  auth: null,
  host: 'api.cloudflare.com',
  port: null,
  hostname: 'api.cloudflare.com',
  hash: null,
  search: null,
  query: undefined,
  pathname: '/client/v4/zones/ZONEID/dns_records',
  href: 'https://api.cloudflare.com/client/v4/zones/ZONEID/dns_records',
  json: true,
  timeout: 10000,
  method: 'POST',
  body: '{"type":"CNAME","name":"SUBDOMAIN","content":"TARGET","ttl":120}',
  headers: 
   { 'user-agent': 'cloudflare/1.0.4 node/5.3.0',
     'accept-encoding': 'gzip,deflate',
     'x-auth-key': 'APIKEY',
     'x-auth-email': 'EMAIL',
     'content-type': 'application/json',
     accept: 'application/json',
     'content-length': 138 },
  agent: undefined,
  followRedirect: true
}
```

Proxied flag is not sent, therefore it doesn't set desired value. API seems to work on it's own.

**Comment by sielay** [comment]
Ok. I think I went to the same issue as #6. It's bit inconsistent behaviour that there is exposed functionality in response and no endpoint to manipulate it.

**Comment by sielay** [comment]
Just got response from your support that `proxied` flag is supported and should be supported. Also it seems not be exposed in official API docs. See support ticket 1194233

**Comment by victoriabernard92** [comment]
HI, Could you please share the exact curl or call you made to generate the request? (excluding your private keys and ID's of course)

**Comment by terinjokes** [comment]
@sielay yeah, I followed up with our WWW team internally after closing #6 and forgot to reopen it.

**Comment by sielay** [comment]
@terinjokes you have PR ready, works for my so far

**Comment by terinjokes** [comment]
Resolved in 1.0.5.
